server/bin/index-pbench

```diff
@@ -2470,10 +3316,12 @@ if __name__ == '__main__':
     parser.set_defaults(tmpdir = os.environ.get("TMPDIR"))
     parsed = parser.parse_args()
 
-    if parsed.dump_index_patterns:
+    if parsed.dump_index_patterns or parsed.dump_templates:
```

Comment:
-I and -Q are mutually exclusive in the implementation: that should be good enough, but I just wanted to make sure that you are OK with that.

Reply:
Well, nothing enforces the mutual exclusion, other than `-I` is processed first, and then `-Q` is checked for.

Reply:
The problem is that if you specify both, only the -I will be active, because in main(), we deal with that first and then return. So I misspoke when I said "muitually exclusive": it's just that "-I -Q" or "-Q -I" is equivalent to "-I".

Reply:
Right, it seems we agree on the existing behavior.  Are you look for a change to this `if` statement to make it check for one and then the other, or just an acknowledgement that it is okay to leave it as is?

Reply:
The latter.

Reply:
Great, for posterity, just noting that we talked and we'll leave it as is for now.